.env

```
@@ -1,5 +1,5 @@
-ELECTRUM_VERSION=4.6.2
-ELECTRUM_CHECKSUM_SHA512=890c1fae4cd2da5f1fea3f07c1c6b537f9557177b86e23c40333ec83743ed46182ec74e43a27416a3eaa9704d87d0a1702d0a789b23467bcc68a17d638b43655
+ELECTRUM_VERSION=4.7.2
+ELECTRUM_CHECKSUM_SHA512=7ecd72df4e2c26c42287ab7a68459dcfd2afe96d5288995f358a5aea51097e4537595209962c3940651d2aa22914961fa2d681f34c7cd19eceaf2a19f9c76282
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ELECTRUM_CHECKSUM_SHA512 key should go before the ELECTRUM_VERSION key

(UnorderedKey)